src/Orc.Controls/Controls/DirectoryPicker/Views/DirectoryPicker.xaml

```diff
@@ -17,20 +17,31 @@
             <ColumnDefinition Width="Auto" />
         </Grid.ColumnDefinitions>
 
-        <Label Grid.Column="0" Content="{Binding LabelText}" Width="{Binding LabelWidth}" />
-        <TextBox Grid.Column="1" IsReadOnly="True" Text="{Binding SelectedDirectory}" />
+        <Label Grid.Column="0" Content="{Binding LabelText}" Width="{Binding LabelWidth}" VerticalAlignment="Center">
+            <Label.Style>
+                <Style TargetType="Label">
+                    <Style.Triggers>
+                            <DataTrigger Binding="{Binding LabelText, Converter={catel:EmptyStringToCollapsingVisibilityConverter}}">
```

Comment:
how does the trigger works without Value property specified? (Same in other cases)

Reply:
I would prefer not to use DataTrigger...why not use Content property and simple Trigger? (but as for me it's not critical as long as it would be deprecated)

Reply:
Another one: what if LabelWidth would be 0 -> what kind of margin it would be?

Reply:
what if LabelWidth = 0... what kind of margin it would be? 0 too?